src/utils.py

```diff
@@ -245,3 +245,19 @@ def get_session_retry(retries=3, backoff_factor=0.2, status_forcelist=(404, 500,
     adapter = HTTPAdapter(max_retries=retry)
     session.mount('http://', adapter)
     return session
+
+
+def is_quickstart_majority(package_list=[]):
+    """Return true if 50% or more packages are from quickstarts.
+
+    param package_list: The list of packages in the payload.
+    """
+    count_quickstart_pck = 0
+    for package_name in package_list:
+        if package_name.startswith(("org.wildfly.swarm",
+                                    "org.springframework.boot",
+                                    "io.vertx")):
+            count_quickstart_pck += 1
```

Comment:
I'd probably export the list of quickstarts to config file/

Reply:
I did not get your point here:confused:

Reply:
Are you saying that we can pass this list as a constant from the config file so that it is easy to modify the list going forward?

Reply:
@tisnik Got it. Thanks.